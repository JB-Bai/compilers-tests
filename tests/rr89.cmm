int VdIRirE(struct rUfKZ6uArNl WN3Xsgk7Oc, float fGLLNlw, int g6gxKGgdg, int aswMBTr, int jJkIEMC6hliF){
	struct ZVVu g9geb9N;
	struct{
	}
	MgOwFEC, tnMVaRfY;
	(HZxW96Mr) - 2273 -(iRtJ6C_2() *((- 098.662E2 - 022507 +(634.236E1) * 00335 /(485.4424) / !((RQs7Tar / 0x295b1 + tEm9AZD || EMHi + 8556.061e-7 * mx1kt7 - ! 5963.7596.mxG8 /(17.774 - UvZWTDaM && 30.4842e3 && 0x7 - 82980[0.4]))) + abUBW * 9.6)));
	86.20 <= 07025;
	while(7995.503e-2) 64.0607[1728.97E1];
}
int y2LlBQu[0xA24][0xcd1][6793], gzPzCaB;
float MGJB7TU, ydjCGDqsV[0x88], ZJF9YMrmK78, PzMr2[0640];
float YGu9k0(float Ctbr, float XgRMo){
	! RMvE_Sj5 = bwCsRn(qhNdX4G7, 736.32) || IE3E9S / 0xF18d;
}

