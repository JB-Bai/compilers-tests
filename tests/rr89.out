Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: int
      FunDec (1)
        ID: VdIRirE
        LP
        VarList (1)
          ParamDec (1)
            Specifier (1)
              StructSpecifier (1)
                STRUCT
                Tag (1)
                  ID: rUfKZ6uArNl
            VarDec (1)
              ID: WN3Xsgk7Oc
          COMMA
          VarList (1)
            ParamDec (1)
              Specifier (1)
                TYPE: float
              VarDec (1)
                ID: fGLLNlw
            COMMA
            VarList (1)
              ParamDec (1)
                Specifier (1)
                  TYPE: int
                VarDec (1)
                  ID: g6gxKGgdg
              COMMA
              VarList (1)
                ParamDec (1)
                  Specifier (1)
                    TYPE: int
                  VarDec (1)
                    ID: aswMBTr
                COMMA
                VarList (1)
                  ParamDec (1)
                    Specifier (1)
                      TYPE: int
                    VarDec (1)
                      ID: jJkIEMC6hliF
        RP
      CompSt (1)
        LC
        DefList (2)
          Def (2)
            Specifier (2)
              StructSpecifier (2)
                STRUCT
                Tag (2)
                  ID: ZVVu
            DecList (2)
              Dec (2)
                VarDec (2)
                  ID: g9geb9N
            SEMI
          DefList (3)
            Def (3)
              Specifier (3)
                StructSpecifier (3)
                  STRUCT
                  LC
                  RC
              DecList (5)
                Dec (5)
                  VarDec (5)
                    ID: MgOwFEC
                COMMA
                DecList (5)
                  Dec (5)
                    VarDec (5)
                      ID: tnMVaRfY
              SEMI
        StmtList (6)
          Stmt (6)
            Exp (6)
              Exp (6)
                Exp (6)
                  LP
                  Exp (6)
                    ID: HZxW96Mr
                  RP
                MINUS
                Exp (6)
                  INT: 2273
              MINUS
              Exp (6)
                LP
                Exp (6)
                  Exp (6)
                    ID: iRtJ6C_2
                    LP
                    RP
                  STAR
                  Exp (6)
                    LP
                    Exp (6)
                      LP
                      Exp (6)
                        Exp (6)
                          Exp (6)
                            Exp (6)
                              MINUS
                              Exp (6)
                                FLOAT: 9866.200195
                            MINUS
                            Exp (6)
                              INT: 9543
                          PLUS
                          Exp (6)
                            Exp (6)
                              Exp (6)
                                Exp (6)
                                  LP
                                  Exp (6)
                                    FLOAT: 6342.359863
                                  RP
                                STAR
                                Exp (6)
                                  INT: 221
                              DIV
                              Exp (6)
                                LP
                                Exp (6)
                                  FLOAT: 485.442413
                                RP
                            DIV
                            Exp (6)
                              NOT
                              Exp (6)
                                LP
                                Exp (6)
                                  LP
                                  Exp (6)
                                    Exp (6)
                                      Exp (6)
                                        Exp (6)
                                          ID: RQs7Tar
                                        DIV
                                        Exp (6)
                                          INT: 169393
                                      PLUS
                                      Exp (6)
                                        ID: tEm9AZD
                                    OR
                                    Exp (6)
                                      Exp (6)
                                        Exp (6)
                                          ID: EMHi
                                        PLUS
                                        Exp (6)
                                          Exp (6)
                                            FLOAT: 0.000856
                                          STAR
                                          Exp (6)
                                            ID: mx1kt7
                                      MINUS
                                      Exp (6)
                                        Exp (6)
                                          NOT
                                          Exp (6)
                                            Exp (6)
                                              FLOAT: 5963.759766
                                            DOT
                                            ID: mxG8
                                        DIV
                                        Exp (6)
                                          LP
                                          Exp (6)
                                            Exp (6)
                                              Exp (6)
                                                Exp (6)
                                                  FLOAT: 17.774000
                                                MINUS
                                                Exp (6)
                                                  ID: UvZWTDaM
                                              AND
                                              Exp (6)
                                                FLOAT: 30484.199219
                                            AND
                                            Exp (6)
                                              Exp (6)
                                                INT: 7
                                              MINUS
                                              Exp (6)
                                                Exp (6)
                                                  INT: 82980
                                                LB
                                                Exp (6)
                                                  FLOAT: 0.400000
                                                RB
                                          RP
                                  RP
                                RP
                        PLUS
                        Exp (6)
                          Exp (6)
                            ID: abUBW
                          STAR
                          Exp (6)
                            FLOAT: 9.600000
                      RP
                    RP
                RP
            SEMI
          StmtList (7)
            Stmt (7)
              Exp (7)
                Exp (7)
                  FLOAT: 86.199997
                RELOP
                Exp (7)
                  INT: 3605
              SEMI
            StmtList (8)
              Stmt (8)
                WHILE
                LP
                Exp (8)
                  FLOAT: 79.955032
                RP
                Stmt (8)
                  Exp (8)
                    Exp (8)
                      FLOAT: 64.060699
                    LB
                    Exp (8)
                      FLOAT: 17289.699219
                    RB
                  SEMI
        RC
    ExtDefList (10)
      ExtDef (10)
        Specifier (10)
          TYPE: int
        ExtDecList (10)
          VarDec (10)
            VarDec (10)
              VarDec (10)
                VarDec (10)
                  ID: y2LlBQu
                LB
                INT: 2596
                RB
              LB
              INT: 3281
              RB
            LB
            INT: 6793
            RB
          COMMA
          ExtDecList (10)
            VarDec (10)
              ID: gzPzCaB
        SEMI
      ExtDefList (11)
        ExtDef (11)
          Specifier (11)
            TYPE: float
          ExtDecList (11)
            VarDec (11)
              ID: MGJB7TU
            COMMA
            ExtDecList (11)
              VarDec (11)
                VarDec (11)
                  ID: ydjCGDqsV
                LB
                INT: 136
                RB
              COMMA
              ExtDecList (11)
                VarDec (11)
                  ID: ZJF9YMrmK78
                COMMA
                ExtDecList (11)
                  VarDec (11)
                    VarDec (11)
                      ID: PzMr2
                    LB
                    INT: 416
                    RB
          SEMI
        ExtDefList (12)
          ExtDef (12)
            Specifier (12)
              TYPE: float
            FunDec (12)
              ID: YGu9k0
              LP
              VarList (12)
                ParamDec (12)
                  Specifier (12)
                    TYPE: float
                  VarDec (12)
                    ID: Ctbr
                COMMA
                VarList (12)
                  ParamDec (12)
                    Specifier (12)
                      TYPE: float
                    VarDec (12)
                      ID: XgRMo
              RP
            CompSt (12)
              LC
              StmtList (13)
                Stmt (13)
                  Exp (13)
                    Exp (13)
                      NOT
                      Exp (13)
                        ID: RMvE_Sj5
                    ASSIGNOP
                    Exp (13)
                      Exp (13)
                        ID: bwCsRn
                        LP
                        Args (13)
                          Exp (13)
                            ID: qhNdX4G7
                          COMMA
                          Args (13)
                            Exp (13)
                              FLOAT: 736.320007
                        RP
                      OR
                      Exp (13)
                        Exp (13)
                          ID: IE3E9S
                        DIV
                        Exp (13)
                          INT: 61837
                  SEMI
              RC
